### PR TITLE
fix ansible xccdf sub

### PR DESCRIPTION
**Pull request by ybznek**
`<sub>` element wasn't included well in ansible remediation.

This PR is not so large, there is only large indentation fix in last commit.

### CODE REVIEW

**Comment by jan-cerny** [comment]
Is that indentation fix necessary in this pull request?

**Comment by jan-cerny** [comment]
The rest is OK in my opinion.

File: shared/transforms/combineremediations.py [hunk 1]
```diff
@@ -208,9 +208,10 @@ def expand_xccdf_subs(fix, remediation_type, remediation_functions):
             xccdfvarsub.tail = text_between_vars
         return
 
+    elif remediation_type == "bash":
     # This remediation script doesn't utilize internal remediation functions
     # Skip it without any further processing
-    elif 'remediation_functions' not in fix.text:
+    if 'remediation_functions' not in fix.text:
```
Comment (mpreisler) [inline shared/transforms/combineremediations.py]: This is invalid python syntax. This commit breaks the script.

**Comment by mpreisler** [comment]
It's not really an indentation fix, it fixes logic of the previous commit. The previous commit looks like a partial change that puts the script in a state where it isn't valid Python2 or 3.

**Comment by ybznek** [comment]
@mpreisler I will squash it. Reason of two separated commits was, that it made huge diff for very small change. I wanted to have more visible changes.

**Comment by ybznek** [comment]
It is already squashed

**Comment by mpreisler** [comment]
ACK

**Pull request merged**